### PR TITLE
Vue modals migration

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR migrate modals from bootstrap.js to Vue.js and move the contribute modal from the gouvfr theme to the core.

screenshot-www data dev 2017-04-18 14-33-31

Alos fix a missing default value on membership role

### CODE REVIEW

File: js/components/organization/member-modal.vue [hunk 1]
```diff
@@ -62,6 +62,7 @@ export default {
         org: Organization
     },
     data() {
+        console.log('API Member', API.definitions.Member);
```
Comment (davidbgk) [inline js/components/organization/member-modal.vue]: To be removed?

**Comment by davidbgk** [comment]
Can you provide a screenshot of the user modal too?

File: js/components/organization/member-modal.vue (hunk 1 above)
Comment (noirbizarre) [inline js/components/organization/member-modal.vue]: Yes, thx

**Comment by noirbizarre** [comment]
It's unchanged, but fixed:
- properly display only the right actions
- default value is set
screenshot-www data dev 2017-04-18 14-47-02
screenshot-www data dev 2017-04-18 14-46-10

**Comment by noirbizarre** [comment]
Don' t forget to review the gouvfr part ;)

**Pull request merged**